server/serverQueryManager.py

```diff
@@ -21,66 +22,64 @@ def add(self, args: dict) -> dict:
         datetime_obj = string_to_datetime(args['date'])
         results = []
         uids_on_day = self.schedule.get(datetime_obj)
-        response_code = 3
+        response_code = ResponseCode.FORBIDDEN
 
         for uid in uids_on_day:
-            if self.adjmat.is_adjacent(args['uid'], uid) == True:
+            if self.adjmat.is_adjacent(args['uid'], uid):
                 results.append(uid)
 
```

Comment:
We need to add a check to make sure that the uid is stored somewhere, and respond that the uid is not recognized if not.